app/src/main/java/com/hammer/talkbbokki/presentation/detail/DetailScreen.kt

```diff
@@ -47,6 +45,32 @@ fun DetailRoute(
     val item by viewModel.item.collectAsState()
     val starter by viewModel.talkOrder.collectAsState()
 
+    var showDialog by remember { mutableStateOf(false) }
+    if (showDialog) {
+        Dialog(onDismissRequest = {}) {
+            Box(modifier = Modifier
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)